scipy/stats/tests/test_stats.py

```diff
@@ -5155,6 +5180,15 @@ def convert(t, p, alt):
     assert_allclose(p, converter(tr, pr, 'greater'), rtol=1e-14)
 
 
+def test_ttest_ind_scalar():
+    # # test scalars
```

Comment:
```suggestion
    # test scalars
```
Copy-pasted out of `test_ttest_ind`.